credentials/tls.go

```diff
@@ -25,14 +25,16 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net"
+	"net/url"
 
 	"google.golang.org/grpc/credentials/internal"
 )
 
 // TLSInfo contains the auth information for a TLS authenticated connection.
 // It implements the AuthInfo interface.
 type TLSInfo struct {
-	State tls.ConnectionState
+	State    tls.ConnectionState
+	SpiffeID *url.URL
```

Comment:
Make this EXPERIMENTAL for now?

Reply:
I noticed that the field `CommonAuthInfo` is already  EXPERIMENTAL so I moved SPIFFEID into it directly. Let me know if it works, thanks!